perfkitbenchmarker/linux_packages/cudnn.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Module containing CUDA Deep Neural Network library installation functions."""
+
+import os
+from perfkitbenchmarker import data
+from perfkitbenchmarker import flags
+
+flags.DEFINE_string('cudnn', 'cudnn-8.0-linux-x64-v5.1.tgz',
+                    '''The NVIDIA CUDA Deep Neural Network library.
+                    Please put in data directory and specify the name''')
+FLAGS = flags.FLAGS
+CUDA_TOOLKIT_INSTALL_DIR = '/usr/local/cuda'
```

Comment:
The you need to add `from perfkitbenchmarker.linux_packages import cuda_toolkit_8`